content/Foundation/Summit/2023/home.pml

```diff
@@ -17,21 +17,22 @@ Minneapolis, USA
 <div class="summit_div">
 <div class="highlight">
 <h4></h4>
-<h4>Latest News: <a href="#summit-sponsors">Sponsors</a></h4>
+<h4>Latest News: <a href="#summit-abstracts">Videos, Slides and Abstracts</a></h4>
```

Comment:
I would already remove this "Latest News" line. This was the page is ready to stay the way it is